.env.example

```
@@ -1,8 +1,18 @@
-# .env.example
+# ── OpenDota API ────────────────────────────────────────────────
 OPENDOTA_BASE_URL=https://api.opendota.com/api
-OPENDOTA_RATE_LIMIT=60
-OPENDOTA_TIMEOUT=30
-OPENDOTA_RETRIES=3
+OPENDOTA_RATE_LIMIT=60    # requests per minute (free tier = 60)
+OPENDOTA_TIMEOUT=30       # HTTP timeout in seconds
+OPENDOTA_RETRIES=3        # retry attempts on 429/5xx
 
+# ── Match Discovery (Epic 2) ─────────────────────────────────────
+DISCOVERY_LOBBY_TYPE=7    # 7 = ranked matchmaking
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)